docs/source/stats.rst

```diff
@@ -125,6 +133,8 @@ Goodness of Fit Tests and Measures
 some tests for goodness of fit for univariate distributions
 
 .. module:: statsmodels.stats.gof
+   :synopsis: Statistical methods and tests measure goodness of fit
```

Comment:
Is alphabetical sorting relevant here?, e.g.
"Goodness of fit measures and tests"
versus
"Statistical methods for goodness of fit measures and tests"